zokrates_cli/src/bin.rs

```diff
@@ -124,20 +124,14 @@ mod tests {
 
         builder
             .spawn(|| {
-                for p in glob("./examples/**/*").expect("Failed to read glob pattern") {
+                for p in glob("./examples/**/*.zok").expect("Failed to read glob pattern") {
```

Comment:
I think there's a danger here: if one makes a typo in the extension of an example it would be skipped silently. It's imo better to crash which I think was the behaviour before this change: ignore folders, force files to have `zok` extension

Reply:
I've introduced a `.sh` file (steps of the tutorial which are being tested in integration tests) in the tutorials which reside in the same folder as the `zok` files, so without filtering, it will fail here because of the wrong extension?

Reply:
Then let's explicitly ignore `.sh`, I guess glob allows that?